content/en/docs/tasks/debug-application-cluster/debug-cluster.md

```diff
@@ -100,11 +100,11 @@ This is an incomplete list of things that could go wrong, and how to adjust your
 - Action: Use IaaS providers reliable storage (e.g. GCE PD or AWS EBS volume) for VMs with apiserver+etcd
   - Mitigates: Apiserver backing storage lost
 
-- Action: Use (experimental) [high-availability](/docs/admin/high-availability) configuration
+- Action: Use [high-availability](/docs/admin/high-availability) configuration
   - Mitigates: Master VM shutdown or master components (scheduler, API server, controller-managing) crashing
```

Comment:
```suggestion
  - Mitigates: Control plane node shutdown or control plane components (scheduler, API server, controller-manager) crashing
```

Reply:
Hello @natereid72. The changes seem okay. Would it be possible for you to incorporate the comments from @justaugustus? Thanks!